### ISSUE TITLE
Visibility expansion on entire network

### ISSUE THREAD

**Issue by psychoet** [jb55/lnvis#14]
Hi, 

Current version of LNVis (v0.1) only shows the limited boundary of the network: only nodes that have channel(s) with my node.
I am not sure if this is intended behavior or not, but is it possible to see entire lightning network from the perspective of my node?

**Comment by jb55** [comment]
>Current version of LNVis (v0.1) only shows the limited boundary of the
>network: only nodes that have channel(s) with my node.

>I am not sure if this is intended behavior or not, but is it possible
>to see entire lightning network from the perspective of my node?

you can definitely view everything but it will be very slow. I have not
optimized this at all. It should only show your local view if you pass a
nodeid filter.

**Comment by psychoet** [comment]
Hmm..I also tried without --filter option, but still only shows my node perspective. Maybe due to the limit on my hardware (intel NUC 11 i5 with 16GB RAM) or there is a default option when there is no --filter argument?
But I can see why this is not recommended. I just changed my strategies to use that filter with other node, and achieved what I wanted to see: if my node is visible from another specific node.

Thanks for making this app!
